core/Cargo.toml

```diff
@@ -34,8 +34,8 @@ std = [
 lock_api = { version = "0.4", features = ["arc_lock"] }
 math = { package = "winter-math", version = "0.9", default-features = false }
 memchr = { version = "2.7", default-features = false }
-#miden-crypto = { version = "0.10", default-features = false }
-miden-crypto = { git = "https://github.com/0xPolygonMiden/crypto", branch = "al-rpo-new-padding-rule", default-features = false }
+miden-crypto = { version = "0.11", default-features = false }
+#miden-crypto = { git = "https://github.com/0xPolygonMiden/crypto", branch = "al-rpo-new-padding-rule", default-features = false }
```

Comment:
Do we still need this line?

Reply:
I wasn't sure, so decided to leave it. @Al-Kindi-0 Is the version of the crypto in your branch the same as v0.11?

Reply:
I think after having merge the related PR in Miden crypto, we can now remove the above.